src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,58 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+};
+
+export const MovieCard: React.FC<Props> = (
+  {
+    title, description, imgUrl, imdbUrl,
+  },
+) => (
+  <>
+    <div className="card" data-cy="Movie">
```

Comment:
```suggestion
    <li className="card" data-cy="Movie">
```
І не забудь замінити закриваючий тег)